src/State/Hub.php

```diff
@@ -239,7 +239,7 @@ public function startTransaction(TransactionContext $context, array $customSampl
             } else {
                 $sampleRate = $this->getSampleRate(
                     $samplingContext->getParentSampled(),
-                    $options->getTracesSampleRate()
+                    $options->getTracesSampleRate() ?? 0
```

Comment:
For future reference: technically we should never be here with a `null` in the sample rate.